src/components/EmployeeModalForm/EmployeeCreateForm.tsx

```diff
@@ -49,25 +49,35 @@ export const EmployeeCreateForm: Component<Props> = ({ state, actions }) => {
             <FormInput
               name={firstName.name}
               value={firstName.value}
+              maxLength={50}
```

Comment:
Pass errors to `FormInput`, `FormSelect`, `DatePicker`, `EmployeeSelect` and `FormCheckbox` and mark the control with `is-danger` if it contains errors.

Reply:
Add constants for 50 => `INPUT_MIN_LENGTH_SHORT = 50`